src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -123,8 +123,8 @@ public async Task<RestoreResult> ExecuteAsync(CancellationToken token)
             {
                 telemetry.TelemetryEvent.AddPiiData(ProjectFilePath, _request.Project.FilePath);
 
-                bool isPackageSourceMappingEnabled = _request?.PackageSourceMapping.IsEnabled ?? false;
-                telemetry.TelemetryEvent[PackageSourceMappingIsMappingEnabled] = isPackageSourceMappingEnabled;
+                bool isPackageSourceMappingAlreadyEnabled = _request.PackageSourceMapping?.IsEnabled ?? false;
+                telemetry.TelemetryEvent[PackageSourceMappingIsMappingEnabled] = isPackageSourceMappingAlreadyEnabled;
```

Comment:
This variable name change doesn't make sense to me. Restore doesn't enable package source mapping half way through restore, so I expect it to be enabled or disabled throughout the entire operation. I don't understand what the "already" in the new variable name is trying to communicate.